src/prpy/tsr/tsr.py

```diff
@@ -122,7 +227,7 @@ def from_json(x, *args, **kw_args):
     def to_yaml(self):
         """ Convert this TSR to a YAML string. """
         import yaml
-        return yaml.dumps(self.to_dict())
+        return yaml.dump(self.to_dict())
```

Comment:
I think we want `dumps` here, since `dump` expects a `File`-like object. What was the motivation behind this change?